src/workers/parser/getSymbols.js

```diff
@@ -44,7 +44,18 @@ export type SymbolDeclarations = {
 
 function getFunctionParameterNames(path: NodePath): string[] {
   if (path.node.params != null) {
-    return path.node.params.map(param => param.name);
+    return path.node.params.map(param => {
+      if (param.name) {
+        return param.name;
+      } else if (
+        // Parameter with default value
+        param.type === "AssignmentPattern" &&
+        param.left.type === "Identifier" &&
+        param.right.type === "Identifier"
+      ) {
+        return `${param.left.name} = ${param.right.name}`;
+      }
+    });
```

Comment:
is there now a case that we're not handling?

Reply:
Yeah. Eg objects, arrays or functions as default values.
Right now I have no idea how these should be displayed

Reply:
I'll try to sketch out how to display the other cases over the weekend